example/src/either/overview.dart

```diff
@@ -1,21 +1,20 @@
 import 'package:fpdart/fpdart.dart';
 
 /// Don't do that! ⚠
-double divideI(int x, int y) {
-  if (y == 0) {
-    throw Exception('Cannot divide by 0!');
-  }
+int divideI(int x, int y) => x ~/ y; // this will throw if y == 0
```

Comment:
Could you raise the comment above the function as a doc comment (`///`)?

Reply:
Done